ci-operator/templates/openshift/installer/cluster-launch-installer-custom-test-image.yaml

```diff
@@ -342,7 +342,7 @@ objects:
       terminationMessagePolicy: FallbackToLogsOnError
       volumeMounts:
       - name: shared-tmp
-        mountPath: /tmp/shared
+        mountPath: /tmp
```

Comment:
I would really prefer to change it to `/tmp/shared` everywhere in the template for consistency?